addons/mail/static/src/web/messaging_menu/messaging_menu.xml

```diff
@@ -21,9 +21,9 @@
         'o-mail-MessagingMenu-header border-bottom d-flex flex-shrink-0 p-2': !env.inDiscussApp,
         }">
             <t t-if="!store.isSmall">
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'all' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'all'">All</button>
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'chat' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'chat'">Chats</button>
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'channel' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'channel'">Channels</button>
+                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'all' ? 'fw-bolder' : ''" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'all'">All</button>
```

Comment:
Maybe we want hovering and active effects on those buttons as well since the `ChatWindow` `Discuss` related buttons all have that. And seems that will make the text more clear to click.

Reply:
I will do this as follow-up PR. The intent of most visual changes in this PR is to make it less harsh, e.g. by using `bg-200` at most and avoid `bg-300`.